docs/getting-started/application/index.rst

```diff
@@ -151,39 +237,73 @@ when the actor is an instance of a ``User``:
     :language: polar
     :lines: 6-7
 
+.. tabs::
+    .. group-tab:: Python
 
-.. code-block:: console
+        .. code-block:: console
+
+            $ curl localhost:5000/whoami
+            <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+            <title>403 Forbidden</title>
+            <h1>Forbidden</h1>
+            <p>Not Authorized!</p>
+
+            $ curl -H "user: alice@foo.com"  localhost:5000/whoami
+            You are alice@foo.com, the CEO at Foo Industries. (User ID: 1)
 
-    $ curl localhost:5000/whoami
-    <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
-    <title>403 Forbidden</title>
-    <h1>Forbidden</h1>
-    <p>Not Authorized!</p>
+    .. group-tab:: Java
 
-    $ curl -H "user: alice@foo.com"  localhost:5000/whoami
-    You are alice@foo.com, the CEO at Foo Industries. (User ID: 1)
+        .. code-block:: console
+
+            $ curl -i localhost:5000/whoami
+            HTTP/1.1 401
+
+            $ curl -H "user: alice@foo.com"  localhost:5000/whoami
+            You are alice@foo.com, the CEO at Foo Industries. (User ID: 1)
 
 .. tip::
 
     Interested in understanding more about what is happening here?
     Check out the :doc:`/using/examples/user_types` example.
 
-The inputs to the ``is_allowed`` call are the current user, the HTTP method,
-and the HTTP request. This information can often be enough to cover a large
-number of uses. For example, if we know that some paths should only
-be accessed by certain roles, we can certainly check for this at this point.
+.. tabs::
+    .. group-tab:: Python
+
+        The inputs to the ``is_allowed`` call are the current user, the HTTP method,
+        and the HTTP request. This information can often be enough to cover a large
+        number of uses. For example, if we know that some paths should only
+        be accessed by certain roles, we can certainly check for this at this point.
+
+    .. group-tab:: Java
+
+        The inputs to the ``isAllowed`` call are the current user, the HTTP method,
```

Comment:
Not for this PR. But would really nice a better way to have an inline literal change between languages without needing a new tab block

Reply:
agreed